contracts/PBAB+Collabs/iyk/GenArt721CoreV2_IYK.sol

```diff
@@ -0,0 +1,195 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "../GenArt721CoreV2_PBAB.sol";
+
+import "@openzeppelin-4.5/contracts/utils/cryptography/ECDSA.sol";
+import "./ISignVerifierRegistry.sol";
+
+pragma solidity 0.8.9;
+
+/**
+ * @title Powered by Art Blocks ERC-721 core contract.
+ * Support for IYK pull mechanism to transfer tokens based on verified signatures
```

Comment:
Would suggest that it is worth having a documentation comment at the top of this file that clarifies that due to this transfer mechanism, these tokens will not be trade-able via secondary marketplaces.